.changeset/deprecate-actionlistv1-promote-actionlistv2.md

```diff
@@ -0,0 +1,152 @@
+---
+'@primer/react': major
+---
+
+### ActionList
+
+ActionList now ships a composable API.
```

Comment:
Similar to [earlier comment](https://github.com/primer/react/pull/1893#discussion_r814901047), I think we should rephrase this to say the previous ActionList has been superseded and has other benefits, as a way to encourage people to upgrade sooner.